Cairo Desktop/CairoDesktop.Configuration/app.config

```diff
@@ -103,6 +103,9 @@
             <setting name="FullWidthTaskBar" serializeAs="String">
                 <value>False</value>
             </setting>
+            <setting name="RunAtLogOn" serializeAs="String">
```

Comment:
Since this is no longer being stored in the settings but rather checked via the registry, can this `<setting>` be removed from the project files?